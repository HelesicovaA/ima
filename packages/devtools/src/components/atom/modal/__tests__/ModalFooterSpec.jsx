import React from 'react';
import { shallow } from 'enzyme';
import ModalFooter from '../ModalFooter';

describe('modalFooter atom', () => {
  let wrapper = shallow(<ModalFooter>Footer</ModalFooter>);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.text()).toBe('Footer');
  });
});
