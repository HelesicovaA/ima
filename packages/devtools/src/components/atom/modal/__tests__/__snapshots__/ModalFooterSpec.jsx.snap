// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter atom should match snapshot 1`] = `
<div
  className="footer"
>
  Footer
</div>
`;
